### PR TITLE
Moving codeowners back to root. Trial 5

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<!--
-Please see https://istio.io/help and if you are a user of Istio, please file issues in
-https://github.com/istio/issues/issues instead of here.
-
-Only confirmed, triaged and labelled issues should be filed here.
-Please add the correct labels and epics (and priority and milestones if you have that information)
--->
```
Comment (ldemailly) [inline .github/issue_template.md]: why would we remove this, it's useful

Comment (rshriram) [reply .github/issue_template.md]: once I get codeowners working, I can put it back. I am all out of ideas. IT works perfectly in istio/api where there is no .github folder. it is not working for this repo.

Comment (ldemailly) [reply .github/issue_template.md]: what's the error/symptom when that directory exist ?

we do get already too many user issues filled here and this comment showing up in
https://github.com/istio/istio/issues/new

is important

**Pull request merged**